playbooks/DeploySDDCMgr.yml

```diff
@@ -0,0 +1,178 @@
+                                     SDDCMgr vcf Password: {{ Nested_SDDCMgr.Credential.vcf.Password }}
+                                      SDDCMgr FIPS_Enable: {{ Nested_SDDCMgr.Component.Appliance.FIPSEnable }}
+
+                                       SDDCMgr Media Path: {{ Deploy.Software.SDDCMgr.Directory }}
+                                       SDDCMgr Media File: {{ Deploy.Software.SDDCMgr.File }}
+
+          =================================================================================================
+      when:
+        - DEBUG.DisplayVariables
+
+    - name: Check if SDDCMgr is already installed
+      ansible.builtin.uri:
+        url: https://{{ Nested_SDDCMgr.Component.Appliance.FQDN }}
+        validate_certs: "{{ Common.PKI.ValidateCerts }}"
+        timeout: 5
+      register: sddcmgr_check
+      ignore_errors: true
+      when:
+        - Deploy.Product.SDDCMgr.Deploy
+
+    - name: Deploy SDDCMgr from OVA
+      community.vmware.vmware_deploy_ovf:
+        hostname: "{{ Target.FQDN }}"
+        username: "{{ Target.User }}"
+        password: "{{ Target.Password }}"
+        validate_certs: "{{ Common.PKI.ValidateCerts }}"
+        datastore: "{{ Target.Datastore }}"
+        allow_duplicates: false
+        # For vCenter deployment, VM Folder path must be the FULL path: /<DataCenter>/vm/<ParentFolder>/<ChildFolder>
+        folder: "{% if Target.Deployment == 'vCenter' %}/{{ Target.DataCenter }}/vm/{{ Target.VMFolder }}\
+                 {% else %}{{ Target.VMFolder }}{% endif %}"
+        datacenter: "{{ Target.DataCenter }}"
+        cluster: "{{ Target.Cluster }}"
+        name: "{{ Nested_SDDCMgr.Component.Appliance.VMName }}"
+        ovf: "{{ Deploy.Software.SDDCMgr.Directory }}/{{ Deploy.Software.SDDCMgr.File }}"
+        networks:
+          "Management": "{{ Target.PortGroup.Management.Name }}"
+        disk_provisioning: "{{ Common.DiskProvisioning }}"
+        power_on: true
+        wait_for_ip_address: true
+        inject_ovf_env: true
+        properties:
+          ROOT_PASSWORD: "{{ Nested_SDDCMgr.Credential.root.Password }}"
+          VCF_PASSWORD: "{{ Nested_SDDCMgr.Credential.vcf.Password }}"
+          BASIC_AUTH_PASSWORD: "{{ Nested_SDDCMgr.Credential.admin.Password }}"
+          BACKUP_PASSWORD: "{{ Nested_SDDCMgr.Credential.backup.Password }}"
+          LOCAL_USER_PASSWORD: "{{ Nested_SDDCMgr.Credential.local.Password }}"
+          FIPS_ENABLE: "{{ Nested_SDDCMgr.Component.Appliance.FIPSEnable }}"
+          vami.hostname: "{{ Nested_SDDCMgr.Component.Appliance.FQDN }}"
+          guestinfo.ntp: "{{ Common.NTP.Server1.IPv4 }}"
+          vami.ip0.SDDC-Manager: "{{ Nested_SDDCMgr.Component.Appliance.Address.IPv4.Address }}"
+          vami.netmask0.SDDC-Manager:  "{{ (Nested_SDDCMgr.Component.Appliance.Address.IPv4.Address + '/' + Nested_SDDCMgr.Component.Appliance.Address.IPv4.Prefix) | ansible.utils.ipaddr('netmask') }}"
```

Comment:
Ansible lint complains about the extra space after colon.

Reply:
Had to l;earn what ansible lint was... neat tool, alerts addressed.